src/main/java/com/epam/ta/reportportal/job/PluginLoaderService.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.ta.reportportal.job;
+
+import com.epam.ta.reportportal.core.integration.plugin.PluginInfo;
+import com.epam.ta.reportportal.entity.integration.IntegrationType;
+
+import java.util.List;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+public interface PluginLoaderService {
+
+	List<PluginInfo> getAllPluginsInfo(List<IntegrationType> integrationTypes);
+
+	List<PluginInfo> getNotLoadedPluginsInfo(List<IntegrationType> integrationTypes);
```

Comment:
what does it mean - "notLoaded"?
Can we comment out all interfaces/their methods?

Reply:
renamed. Java docs were provided.